restaurant/src/app/admin/view-sold-foods/view-sold-foods.component.html

```diff
@@ -0,0 +1,19 @@
+<h4 class="text-center"> Enter the dates to find the count </h4>
+<div class="container">
+<div class="col-xs-3">
+<form method="get"  #viewFoods = ngForm   (ngSubmit)="onSubmit()">
+    <div class="form-group" class="col-xs-3">
+        <label for="FromDate">Enter the Start Date</label>
+        <input type="date" id = "from"  class="form-control" >
+    </div>
+    <div class="form-group" class="col-xs-3">
+        <label for="todate">Enter the End Date</label>
```

Comment:
give appropriate value for label field. refer 
```suggestion
        <label for="toDate">Enter the End Date</label>
        <input type="date" id="toDate" class="form-control" >
```